content/articles/getting-hands-on-on-a-microprofile-project/index.md

```diff
@@ -0,0 +1,416 @@
+MicroProfile specifies a collection of Java EE (Enterprise) technologies and APIs which in consolidation from a core baseline microservice that aims to deliver application portability across multiple MicroProfile runtimes.
```

Comment:
Change "...which in consolidation..." to "..that consolidate..."

Reply:
Ok

Reply:
Done! 🚀